manager_redis.go

```diff
@@ -0,0 +1,108 @@
+package ladon
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+	"gopkg.in/redis.v5"
+)
+
+// RedisManager is a redis implementation of Manager to store policies persistently.
+type RedisManager struct {
+	db        *redis.Client
+	keyPrefix string
+}
+
+// NewRedisManager initializes a new RedisManager with no policies
+func NewRedisManager(db *redis.Client, keyPrefix string) *RedisManager {
+	return &RedisManager{
+		db:        db,
+		keyPrefix: keyPrefix,
+	}
+}
+
+const redisPolicies = "ladon:policies"
+
+func (m *RedisManager) redisPoliciesKey() string {
+	return m.keyPrefix + redisPolicies
+}
+
+// Create a new policy to RedisManager
+func (m *RedisManager) Create(policy Policy) error {
+	payload, err := json.Marshal(policy)
+	if err != nil {
+		return err
+	}
+
+	keySet, err := m.db.HSetNX(m.redisPoliciesKey(), policy.GetID(), string(payload)).Result()
+	if !keySet {
+		return errors.New("Policy exists")
+	} else if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (m *RedisManager) getPolicyFromRedis(id string) (Policy, error) {
+	resp, err := m.db.HGet(m.redisPoliciesKey(), id).Bytes()
+	if err == redis.Nil {
+		return nil, errors.New("Not found")
```

Comment:
I think we should have a global var for it `var redisNotFound = errors.New("Not found")`